### PR TITLE
Add deploy steps to Travis

### PR DESCRIPTION
**Pull request by nex3**
This doesn't add Homebrew support yet. Homebrew releases are easiest
to do as edits of an existing Homebrew formula, and creating one of
those requires at least one existing release.

Partially addresses #26

### CODE REVIEW

**Comment by nex3** [comment]
@jathak I copied this logic from Dart Sass; please make sure I didn't miss any instances where I should have replaced "Sass" with "Sass Migrator" or something similar.

File: package/sass-migrator.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+# Copyright 2019 Google Inc. Use of this source code is governed by an MIT-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/MIT.
+
+# This script drives the standalone Sass package, which bundles together a Dart
+# executable and a snapshot of Sass. It can be created with `pub run grinder
```
Comment (jathak) [inline package/sass-migrator.sh]: Sass -> Sass Migrator (here and on line above)

File: tool/grind/chocolatey.dart [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+  request.headers["X-NuGet-ApiKey"] = environment("CHOCO_TOKEN");
+  request.files.add(await http.MultipartFile.fromPath(
+      "package", "build/sass-migrator.${_chocolateyVersion()}.nupkg"));
+
+  var response = await request.send();
+  if (response.statusCode ~/ 100 != 2) {
+    fail("${response.statusCode} error creating release:\n"
+        "${await response.stream.bytesToString()}");
+  } else {
+    log("Released Dart Sass ${_chocolateyVersion()} to Chocolatey.");
```
Comment (jathak) [inline tool/grind/chocolatey.dart]: Dart Sass -> Sass Migrator

File: tool/grind/standalone.dart [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+      vmArgs: ['--snapshot=build/sass_migrator.dart.snapshot']);
+}
+
+@Task('Build a dev-mode Dart application snapshot.')
+appSnapshot() => _appSnapshot(release: false);
+
+@Task('Build a release-mode Dart application snapshot.')
+releaseAppSnapshot() => _appSnapshot(release: true);
+
+/// Compiles Sass to an application snapshot.
```
Comment (jathak) [inline tool/grind/standalone.dart]: Sass -> Sass Migrator

File: tool/grind/standalone.dart [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+package() async {
+  var client = http.Client();
+  await Future.wait(["linux", "macos", "windows"].expand((os) => [
+        _buildPackage(client, os, x64: true),
+        _buildPackage(client, os, x64: false)
+      ]));
+  client.close();
+}
+
+/// Builds a standalone Sass package for the given [os] and architecture.
```
Comment (jathak) [inline tool/grind/standalone.dart]: Sass -> Sass Migrator

File: package/sass-migrator.bat [hunk 5]
```diff
@@ -0,0 +1,12 @@
+@echo off
+REM Copyright 2016 Google Inc. Use of this source code is governed by an
+REM MIT-style license that can be found in the LICENSE file or at
+REM https://opensource.org/licenses/MIT.
+
+REM This script drives the standalone Sass package, which bundles together a
+REM Dart executable and a snapshot of Sass. It can be created with `pub run
```
Comment (jathak) [inline package/sass-migrator.bat]: Sass -> Sass Migrator (here and on line above)

**Pull request merged**